sdk/iotfirmwaredefense/arm-iotfirmwaredefense/tsp-location.yaml

```diff
@@ -0,0 +1,4 @@
+directory: specification/fist/IotFirmwareDefense.Management
+commit: 4ae718636a7debbed8a9c0b790fdbfcd7ee05653
```

Comment:
pending for this pr merge - https://github.com/Azure/azure-rest-api-specs/pull/34201